org.wabarc.wayback.yml

```
@@ -0,0 +1,36 @@
+app-id: org.wabarc.wayback
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+  env:
+    GOBIN: /app/bin
+command: wayback
+finish-args:
+  - --share=network
+  - --filesystem=xdg-documents
+modules:
+  - name: wayback
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/golang/enable.sh
+        export GOPATH=$PWD/go
+        go version
+        make build
+      - install -Dm755 -t /app/bin build/binary/wayback
+    sources:
+      - type: git
+        url: https://github.com/wabarc/wayback.git
+        tag: v0.18.1
+        commit: 97b22585cb5d8ec2417e8449351a9d168a350d97
+      - type: file
+        path: "org.wabarc.wayback.appdata.xml"
+      - type: file
+        path: wayback_128x128.png
+    post-install:
+      - install -Dm644 org.wabarc.wayback.appdata.xml /app/share/appdata/org.wabarc.wayback.appdata.xml
+      - install -Dm644 wayback_128x128.png /app/share/app-info/icons/flatpak/128x128/org.wabarc.wayback.png
```

you install icons in an invalid locaiton

```suggestion
      - install -Dm644 wayback_128x128.png /app/share/icons/hicolor/128x128/apps/org.wabarc.wayback.png
```